staking_deposit/credentials.py

```diff
@@ -139,7 +139,7 @@ def deposit_datum_dict(self) -> Dict[str, bytes]:
         datum_dict.update({'deposit_message_root': self.deposit_message.hash_tree_root})
         datum_dict.update({'deposit_data_root': signed_deposit_datum.hash_tree_root})
         datum_dict.update({'fork_version': self.chain_setting.GENESIS_FORK_VERSION})
-        datum_dict.update({'eth2_network_name': self.chain_setting.ETH2_NETWORK_NAME})
+        datum_dict.update({'NETWORK_NAME': self.chain_setting.NETWORK_NAME})
```

Comment:
any reason for making `'NETWORK_NAME'` in uppercase? or should we change it to lowercase:

```suggestion
        datum_dict.update({'network_name': self.chain_setting.NETWORK_NAME})
```